WRITING-STYLE.md

```diff
@@ -344,6 +344,8 @@ When inserting example code in the text:
 * There should *not* be a space before the colon.
 * Place a blank line after the colon and before the code block.
 
+When substitutions are required in code blocks, for example in requiring the reader to write their own API key, write the text prompt enclosed in a less than sign and a greater than sign. 
```

Comment:
I would put this in a new section at this point, called "replaceable text" or something similar. The reason is we sometimes have inline code with replaceable text - not just in codeblocks.

I think adding a simple example (or two) would be useful too.

Reply:
Yes, good point! The page is quite example-light so I didn't know, but I will add one